docs/modules/contours.md

```diff
@@ -14,10 +14,22 @@ Find and outputs the `contours` of the input `imgBW`. This utilizes the OpenCV m
 
 Finds and outputs the convex hull of each contour in the input `contours`. A convex hull is the convex form of a contour, meaning that the outside angles of each vertex is at least 180 degrees. One way to make this easier to understand is to imagine wrapping a string around the contour. This string would be the convex hull, as all its outside angles would be greater than 180 degrees.
 
+## ContourApproximate
+
+![ContourApproximate Node Image](../assets/images/modules/contours/node_contour_approximate.png)
+
+This node approximates contours by reducing their number of points. It takes one perameter, `epsilon`, which is the maximum distance between the output contour and the input contour. This node works the same way as the OpenCV function [ApproxPolyDP](https://docs.opencv.org/2.4/modules/imgproc/doc/structural_analysis_and_shape_descriptors.html#approxpolydp).
```

Comment:
https://docs.opencv.org/trunk/d3/dc0/group__imgproc__shape.html#ga0012a5fdaea70b8a9970165d98722b4c

Reply:
I change my mind use this instead lol: https://docs.opencv.org/master/d3/dc0/group__imgproc__shape.html#ga0012a5fdaea70b8a9970165d98722b4c